cmds/webboot/webboot_test.go

```diff
@@ -183,6 +183,89 @@ func TestBackOption(t *testing.T) {
 	}
 }
 
+// test data
+var testDistros = map[string]Distro{
+	"FakeDistro": Distro{
+		checksum:     "1234567",
+		checksumType: "sha256",
+	},
+	"FakeDistroNoChecksum": Distro{},
+	"FakeDistroGoodChecksum": Distro{
+		checksum:     "407dc87b95afbe268e760313971041860f36e953a2116db03418a98ce46d61bc",
+		checksumType: "sha256",
+	},
+}
+
+// If the checksum was incorrect and the user does not want to proceed, displayChecksumPrompt should return a DownloadOption
```

Comment:
Convert to table-driven testing. See: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests